ui.frontend/src/utils.js

```diff
@@ -385,6 +390,78 @@ class Utils {
         }
     }
 
+    /**
+     * Registers custom functions from form json.
+     * @param {object} formJson - The Sling model exporter representation of the form
+     */
+    static async registerCustomFunctionsV3(formJson) {
```

Comment:
Can we create a separate utility file specifically for the rule editor, parallel to utils.js? Ideally, this code isn't something customers need to understand, so having a dedicated utility file for the rule editor would help clarify which module owns it.

Reply:
Better naming than V3 and also proper documentation on what is the input/output

Reply:
Updated name as it was no linger being used as a registration function.
I will discuss with Rishi regarding separate util file.